platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/transients/TransientFieldValueTypes.java

```diff
@@ -28,6 +29,11 @@
 		return getComparable(BigDecimal.class);
 	}
 
+	public static final ITransientFieldValueType<DayTime> getDayTime() {
```

Comment:
Wow, this should not have been missing here.

We should add methods (and base classes) for other missing types as well (e.g. `Day`)... but probably not in this PR.

Reply:
I agree.